devito/core/cpu.py

```diff
@@ -316,6 +316,7 @@ def _make_clusters_passes_mapper(cls, **kwargs):
 
         return {
             'blocking': Blocking(options).process,
+            'skewing': Skewing(options).process,
```

Comment:
Eventually, instead of exposing `Blocking(...)` to the pipelines, we could rather expose 

```
def blocking(...)
   ...
```

which calls Skewing (if requested) and Blocking back to back (plus any other routines to achieve temporal blocking?) 

so in the pipelines we'll have 

```
clusters = blocking(clusters)
```

This will make the things a bit more structured.

After all, skewing always goes hand in hand with blocking (when temporal blocking is requested)